docs/node/timeouts.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Timeouts
+sidebar_position: 8
+---
+
+Timeouts can be used to limit the time a server may take to process a response.
+In Connect-ES, timeout values are set by the client via the `timeoutMs` option 
+when issuing a requests. If handling the response takes longer than the timeout, 
+they will respond with the error code `deadline_exceeded`. In gRPC, the concept 
+is also known as [deadlines](https://grpc.io/docs/guides/deadlines/).
```

Comment:
We should let users know that timeouts are also honored by clients. Even if a connection becomes unresponsive, the client call will still abort at the configured timeout.